### PR TITLE
Correct background color styling

### PR DESCRIPTION
**Pull request by usphantomlancer**

### CODE REVIEW

**Review by jpshackelford — CHANGES_REQUESTED** [review]
Please see comments on jps.css.

File: static/css/jps.css [hunk 1]
```diff
@@ -17,6 +17,11 @@
     --background-color: #c0b5a8;
 }
 
+[data-theme=light],
+article:not([data-theme=dark]) {
```
[the remark this answers is not in the document]
Comment (usphantomlancer) [reply static/css/jps.css:21]: Okay. Got it.

[the remark this answers is not in the document]
Comment (usphantomlancer) [reply static/css/jps.css:21]: Hi, John. I have pushed the updates. Please check.

[the remark this answers is not in the document]
Comment (jpshackelford) [reply static/css/jps.css:21]: I've verified. Thanks very much!

**Pull request merged**